generators/Exercises/Generators/Allergies.cs

```diff
@@ -11,19 +14,34 @@ protected override void UpdateTestMethod(TestMethod testMethod)
             testMethod.TestedMethodType = TestedMethodType.InstanceMethod;
 
             if (testMethod.Property == "allergicTo")
+            {
                 testMethod.Assert = RenderIsAllergicToAssert(testMethod);
+            }
             else if (testMethod.Property == "list")
+            {
                 testMethod.UseVariableForExpected = true;
+                testMethod.Expected = ConvertExpected(testMethod.Expected);
+            }
         }
 
         private string RenderIsAllergicToAssert(TestMethod testMethod)
         {
             var assert = new StringBuilder();
 
             foreach (var allergy in testMethod.Expected)
-                assert.AppendLine(Render.AssertBoolean(allergy["result"], $"sut.IsAllergicTo({Render.Object(allergy["substance"])})"));
+                assert.AppendLine(Render.AssertBoolean(allergy["result"], $"sut.IsAllergicTo({RenderAllergen(allergy["substance"])})"));
 
             return assert.ToString();
         }
+
+        private UnescapedValue[] ConvertExpected(dynamic expected)
+        {
+            if (expected is string[] allergens)
+                return allergens.Select(RenderAllergen).ToArray();
+
+            return Array.Empty<UnescapedValue>();
+        }
+
+        private UnescapedValue RenderAllergen(dynamic input) => Render.Enum("Allergen", input);
```

Comment:
Minor: does `nameof(Allergen)` work here?

Reply:
Nope, because the `Allergen` enumeration is not defined in the generators project :)

Reply:
*face palm* yup makes sense :)